tests/xharness/Harness.cs

```diff
@@ -491,7 +491,7 @@ int ConfigureIOS ()
 			foreach (var monoNativeInfo in IOSTestProjects.Where (x => x.MonoNativeInfo != null).Select (x => x.MonoNativeInfo))
 				monoNativeInfo.Convert ();
 
-			foreach (var proj in IOSTestProjects) {
+			foreach (var proj in IOSTestProjects.Where ((v) => v.GenerateVariations).ToArray ()) {
```

Comment:
do you need the `ToArray ()` call here ?
`Where` returns an `IEnumerable<T>` so the `foreach` should be happy with it

Reply:
Fixed; the corresponding mac code adds to MacTestProjects, which is why the ToArray is needed there, but the iOS code doesn't do that, so the ToArray is redundant.